content/en/docs/concepts/architecture/cloud-controller.md

```diff
@@ -107,7 +105,9 @@ routes appropriately. It requires Get access to Node objects.
 
 ### Service controller {#authorization-service-controller}
 
-The service controller listens to Service object Create, Update and Delete events and then configures Endpoints for those Services appropriately (for EndpointSlices, the kube-controller-manager manages these on demand).
+The service controller listens to Service object Create, Update and Delete events and then
+configures Endpoints for those Services appropriately (for EndpointSlices, the
+kube-controller-manager manages these on demand).
```

Comment:
Is this text still true given that we now favor EndpointSlices? I know that one can use Services even without a cloud provider integration.

Reply:
I think this is still true. cloud-controller-manager is not a must have.